### PR TITLE
Add Tabs on Case Dashboard 

### PR DESCRIPTION
**Pull request by jonkiky**
## Description

Add sample and file tabs on case dashboard, without design, just a basic tab function. 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? N/A

### CODE REVIEW

**Review by skiran86 — COMMENTED** [review]
IN .env file can we keep REACT_APP_BACKEND_API as "http://localhost:8080/v1/graphql/"

**Comment by jonkiky** [comment]
> IN .env file can we keep REACT_APP_BACKEND_API as "http://localhost:8080/v1/graphql/"
updated .env file

File: src/pages/dashboardTab/dashboardState.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
+import client from '../../utils/graphqlClient';
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/dashboardState.js:1]: Can you separate Actions and Reducers here too!!!

**Comment by AjayDoddapaneni** [comment]
@jonkiky What's going on with Cadacy?

**Pull request closed without merging**